packages/phone-number-privacy/combiner/test/integration/legacypnp.test.ts

```diff
@@ -12,6 +11,7 @@ import {
   TestUtils,
   WarningMessage,
 } from '@celo/phone-number-privacy-common'
+import { AttestationsStatus } from '@celo/phone-number-privacy-common/lib/test/utils'
```

Comment:
just checking that this is importing from test?

Reply:
yup! (these are normally used from `TestUtils` but was running into type errors trying to import the interface that way)

this file + all of the `AttestationsStatus` stuff should be deleted after the `LEGACY_PNP` endpoint is fully deprecated :)